### PR TITLE
[feature] #2053: Add tests to the asset-related queries in private blockchain

### LINKED ISSUE #2053: Unit test the `private_blockchain` permissions validators
At present we don't have any unit tests for the `private_blockchain` use case. It would be important to include them in the test coverage of the integration tests.

### PR DESCRIPTION
**Pull request by pesterev**
### Description of the Change

Add tests to the asset-related queries in private blockchain. 

### Issue

Partially resolves #2053 .

### Benefits

New tests.

### Possible Drawbacks

None.

### CODE REVIEW

File: permissions_validators/src/private_blockchain/mod.rs [hunk 1]
```diff
@@ -40,3 +40,1002 @@ impl IsGrantAllowed for ProhibitGrant {
...
+                assert!(only_accounts_data.check(&bob_id, &find_gold, &wsv).is_err());
+                assert!(only_accounts_data
+                    .check(&carol_id, &find_gold, &wsv)
+                    .is_err());
+            }
+        }
+    }
+
+    #[test]
+    fn find_assets_by_domain_id() {
```
Comment (appetrosyan) [inline permissions_validators/src/private_blockchain/mod.rs:649]: Is this supposed to be outside `mod queries`?

Comment (appetrosyan) [reply permissions_validators/src/private_blockchain/mod.rs:649]: If so, I'd add an explicit `mod` that explains what these tests are.

Comment (pesterev) [reply permissions_validators/src/private_blockchain/mod.rs]: Ah, that's after the wrong rebase. Thanks, fixed.

**Review by appetrosyan — APPROVED** [review]

File: permissions_validators/src/private_blockchain/mod.rs [hunk 2]
```diff
@@ -572,6 +572,387 @@ mod tests {
...
+        fn find_assets_by_account_id() {
+            let TestEnv {
+                alice_id,
+                bob_id,
+                carol_id,
+                wsv,
+                ..
+            } = TestEnv::new();
+
+            let op = QueryBox::FindAssetsByAccountId(FindAssetsByAccountId::new(alice_id.clone()));
```
[the remark this answers is not in the document]
Comment (pesterev) [reply permissions_validators/src/private_blockchain/mod.rs:586]: if you look at previous tests you will find similar naming. In this test case one query is enough and no matter how it's called.

[the remark this answers is not in the document]
Comment (pesterev) [reply permissions_validators/src/private_blockchain/mod.rs:586]: `op` refers to `operation` here because `check` receive an argument called `operation`.

**Review by s8sato — APPROVED** [review]

File: permissions_validators/src/private_blockchain/mod.rs (hunk 2 above)
[the remark this answers is not in the document]
Comment (appetrosyan) [reply permissions_validators/src/private_blockchain/mod.rs:586]: While I would prefer a more descriptive name in general, in this case the context gives enough information.

[the remark this answers is not in the document]
Comment (Erigara) [reply permissions_validators/src/private_blockchain/mod.rs:586]: ok, it makes sense

[the remark this answers is not in the document]
Comment (appetrosyan) [reply permissions_validators/src/private_blockchain/mod.rs:586]: However, please refer to our contributing guide, @pesterev, we prefer full words as names, with very few abbreviations allowed.

**Pull request merged**